qmstr/qmstr-util/remotelog.go

```diff
@@ -0,0 +1,55 @@
+package util
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type LogMessage struct {
+	Msg string
+}
+
+type HTTPRemoteLogWriter struct {
+	host     string
+	port     int
+	endpoint string
+}
+
+func NewHTTPRemoteLogger(host string, port int, endpoint string) *HTTPRemoteLogWriter {
+	return &HTTPRemoteLogWriter{host, port, endpoint}
+}
+
+func (rlw *HTTPRemoteLogWriter) Write(p []byte) (int, error) {
```

Comment:
This method is never used

Reply:
Of course it is. It actually is what makes remote logging work. It is the implementation of io.Writer so the HTTPRemoteLogWriter can be used as a sink for the logger.

Reply:
You are right, I got it.